### ISSUE TITLE
"Error: 'rp.id' cannot be used with the current origin" when using FIDO2 and 127.0.0.1

### ISSUE THREAD

**Issue by maximbelyayev** [oliwarner/django-multifactor#90]
Receive the following error when attempting to add a FIDO2 device using `FIDO_SERVER_ID = '127.0.0.1`:

```FIDO2 Security Key
Error: 'rp.id' cannot be used with the current origin

[Try again](javascript:void(0)) or [Go to Multifactor Dashboard](https://127.0.0.1:8000/mfa/multifactor/)
```

This is of course on url `https://127.0.0.1:8000/mfa/multifactor/fido2/new/`

**Comment by oliwarner** [comment]
I can't make 127.0.0.1 work but I can make localhost:8000 work (as long as there's a TLS certificate - runserver_plus/Werkzeug helps in dev)

    ./manage.py runserver_plus --cert-file cert.crt

**Comment by oliwarner** [comment]
Closed per #70 

IPs will never work with https and therefore not fido2, but `localhost` can work, as can various other real domain options that you can host locally or through (eg) Cloudflare tunnels.
